src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs

```diff
@@ -152,7 +152,10 @@ public IList<string> Channels
         public bool Continuous
         {
             get => _continuous;
-            set => _freezer.SetValue(ref _continuous, value);
+            set {
+                _freezer.SetValue(ref _continuous, value);
+                Options.MaxRetries = _continuous ? Int32.MaxValue : 9;
```

Comment:
I worry about this.  What if the user does this:

```
Config.MaxRetries = 5;
Config.Continuous = false;
```

Reply:
Issue fixed.

Reply:
But P2P recoverable tests are failing. Not sure if the change has caused the failures. Need to take a look.

Reply:
After set the MaxRetries and MaxRetryWaitTime to the old default value 2 and 10 min, the P2P recoverable failure tests are passed.